.eslintrc

```diff
@@ -1,3 +1,13 @@
 {
-    "extends": "ebay"
-}
\ No newline at end of file
+    "extends": "ebay",
+    "rules": {
+         "import/order": [2, {"groups": ["builtin", "external", "internal", "parent", "sibling", "index"]}],
+        "no-param-reassign": 0
+    },
+    "parserOptions": {
+        "ecmaVersion": 2017,
```

Comment:
`eslint-config-ebay` is currently on version `2.x`. 

ecmaVersion support for `2017` requires `eslint@3.6.0` or higher

https://eslint.org/blog/2016/09/eslint-v3.6.0-released

Can you confirm what `eslint` version you are using locally? And if it's `2.x`, confirm that `"ecmaVersion": 2017` is not throwing parse errors?

Reply:
Also, we reference `"ecmaVersion": 2017` here as well. We can probably just keep it in one place.